test/integration/wallet.test.js

```diff
@@ -192,6 +193,35 @@ describe('@integration: Wallet', () => {
       step('When I click "send button"', () => helper.clickOnElement('.send-button button'));
       step(`Then I should see text ${successMessage} in "result box message" element`, () => helper.haveTextOf('.result-box-message', successMessage));
     });
+
+    describe('Scenario: should show account initialisation option if no public key and balance is greater than 0', () => {
```

Comment:
At least one test should go through the whole process of sending the "Initialization" transaction  (but maybe it can be e2e test)